tools/travis.sh

```diff
@@ -49,7 +52,18 @@ fi
 
 if [ "${OPAM_LINT}" = 1 ]; then
     find packages -iname opam -print | xargs -n 1 opam lint
+elif [ "${CHECK_UNUSED}" = 1 ]; then
+    opam install -y --fake $XS $(pkg xs-extra-dummy)
```

Comment:
This should only be used with `COMPILE_ALL=1`, otherwise it will list the packages that are only needed by xs-extra/